supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -570,6 +620,58 @@ An Ansible task is a set of instructions to achieve a state defined, in its broa
 
 *See also*:  xref:xplat[Microsoft Azure Cross-Platform Command-Line Interface]
 
+[discrete]
+[[ARM64]]
+==== image:images/yes.png[yes] ARM64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform (OCP) attributes, Kubernetes, Operators, APIs, and CLI objects. Use this format in general sentences when referring to OCP features.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Creating an ARM64 compute machine set._ 
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: xref:arm64[arm64]
+
+[discrete]
+[[arm64]]
+==== image:images/yes.png[yes] arm64 (noun)
+*Description*: A 64-bit version of the ARM architecture. Use this term for OpenShift Container Platform attributes, Kubernetes, Operators, APIs, and CLI objects. Use lowercase format and backticks when referring to objects or parameters.
+
+Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
+
+Example: 
+
+_Valid values are `arm64`._
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: c
+
+[discrete]
+[[64-bit ARM]]
```

Comment:
I noticed that the 64-bit ARM and 64-bit x86 entries are not rendering properly in my local preview. I'm not sure if this will be resolved after merge but I wanted to ask someone here s well